CefSharp.Wpf.Example/Handlers/PermissionHandlerExample.cs

```diff
@@ -0,0 +1,34 @@
+using CefSharp.Handler;
```

Comment:
Add License disclaimer

Reply:
As there doesn't seem to be anything `WPF` specific in this example it can be moved to the `CefSharp.Example` project and used for both the `WPF` and `WinForms` examples.